spec/plugin_spec.rb

```diff
@@ -184,6 +188,12 @@ def topic_view
 
     context "regular" do
       it "renders channel" do
```

Comment:
I think we should update this description to say "renders channel, excluding inactive, staged, and suspended users"